src/test/groovy/org/prebid/server/functional/tests/AmpSpec.groovy

```diff
@@ -266,4 +266,32 @@ class AmpSpec extends BaseSpec {
         assert bidderRequest.imp[0]?.banner?.format[0]?.w == ampStoredRequest.imp[0].banner.format[0].w
         assert bidderRequest.regs?.ext?.gdpr == ampStoredRequest.regs.ext.gdpr
     }
+
+    def "PBS should generate UUID for BidRequest id and merge StoredRequest when generate-storedrequest-bidrequest-id = true or id = {{UUID}}"() {
```

Comment:
1) Please don't put `or` in the test description.
2) Your test is parametrized. Please use placeholders to describe actual parameter values.